### PR TITLE
Change column types to allow indices

### PR DESCRIPTION
**Pull request by sebastienros**
With the increase of data points for each new job, and the fact that graphs will need more filters, adding indices should have a significant impact on perf. The columns that need it will require to have more specific limits and get away from the TEXT(nvarchar(max)) type.

### CODE REVIEW

**Comment by sebastienros** [comment]
updated

File: src/BenchmarksDriver/Program.cs [hunk 1]
```diff
@@ -724,9 +724,20 @@ private static Task WriteJobsToSql(ServerJob serverJob, ClientJob clientJob, str
...
                         [Value] [float] NOT NULL
                     )
+
+                    CREATE INDEX IDX_Scenario_Hardware_OS_Description_Dimension
+                    ON AspNetBenchmarks
+                    (
+                        [Scenario],
+                        [Hardware],
+                        [OperatingSystem],
+                        [Description],
```
Comment (mikeharder) [inline src/BenchmarksDriver/Program.cs]: Why index `Description`?

**Comment by mikeharder** [comment]
@sebastienros: Do we know if these indexes improve PowerBI perf?  We should be able to identify the queries used by PowerBI, either via SQL Azure Auditing or another mechanism, and then test the queries before and after the indexes.

**Comment by sebastienros** [comment]
Ok, I will get some numbers.

File: src/BenchmarksDriver/Program.cs (hunk 1 above)
Comment (sebastienros) [reply src/BenchmarksDriver/Program.cs]: This is to get only the data points that are run with the CI job. If we get more automatic data points like 2.0/2.1 comparisons, we don't want these to be in the graphs.

Comment (sebastienros) [reply src/BenchmarksDriver/Program.cs]: BTW I have another branch (not yet PR'ed) to set this description as part of the CI job.

**Pull request closed without merging**